src/rebuild/scan.c

```diff
@@ -1117,21 +1117,31 @@ rebuild_tgt_scan_handler(crt_rpc_t *rpc)
 	 */
 	d_list_for_each_entry(rpt, &rebuild_gst.rg_tgt_tracker_list, rt_list) {
 		if (uuid_compare(rpt->rt_pool_uuid, rsi->rsi_pool_uuid) == 0 &&
-		    ((rpt->rt_rebuild_ver < rsi->rsi_rebuild_ver) ||
-		    (rpt->rt_rebuild_op == rsi->rsi_rebuild_op &&
-		     rpt->rt_rebuild_ver == rsi->rsi_rebuild_ver &&
-		     rpt->rt_rebuild_gen < rsi->rsi_rebuild_gen))) {
```

Comment:
why the rt_rebuild_ver and rt_rebuild_gen comparison removed? thx

Reply:
oh,  it only remove rebuild_gen compare here.  But the generation will be increased on each rebuild schedule, 

so if the PS leader is being changed multiple times, then some rebuild might be aborted incorrectly, for example in one rebuild process.

1. rank1 as PS leader,  rebuild schedule(),  gen 1.
2. rank2 become the new PS leader, then rebuild schedule, gen1.
3. rank1 become the PS leader again. rebuild schedule(),  gen increase to 2.   Though it is still the same rebuild.   

And with this PR, it will only compare the pool map version.